applications/tari_app_grpc/proto/base_node.proto

```diff
@@ -61,6 +61,8 @@ service BaseNode {
     // Fetch any utxos that exist in the main chain
     rpc FetchMatchingUtxos(FetchMatchingUtxosRequest) returns (stream FetchMatchingUtxosResponse);
 
+    // get all peers from the base node
+    rpc GetPeers(Empty) returns (GetPeersResponse);
```

Comment:
Add a GetPeersRequest, even if it's empty. This way we can add arguments without breaking the signature.  Some arguments might be filtering by online, basenode/wallet, supported features. etc.